observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.adoc

```diff
@@ -8,35 +8,19 @@ toc::[]
 
 include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
 
+You can use the {DTShortName} in combination with the xref:../../otel/otel-forwarding.adoc#otel-forwarding-traces[{OTELName}].
```

Comment:
This links to https://73592--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-forwarding#otel-forwarding-traces which throws me off a bit because it doesn't mention "Red Hat build of OpenTelemetry" really other than as a prereq, although that's what the link is called. Maybe it would make more sense instead to link to https://73592--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-installing if there's no overview type page for https://73592--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/otel/otel-installing?

Reply:
Thank you for this feedback. I updated it to the following:
```
You can use the {DTShortName} xref:../../otel/otel-forwarding.adoc#otel-forwarding-traces[in combination with] the xref:../../otel/otel-installing.adoc#install-otel[{OTELName}].
```